### ISSUE TITLE
not rendering everyone sometimes

### ISSUE THREAD

**Issue by zuqrr** [TKazer/CS2_External#144]
randomly people disappear from the esp

**Comment by jhonthedoe** [comment]
Yes same problem here

**Comment by MrWaffleManTheGreat** [comment]
my esp doesnt even work

**Comment by pipi-ly** [comment]
The offsets will not be updated in the future, please update it by self.

**Comment by jhonthedoe** [comment]
We dont say we wont update it on future.
Even a2x did not released anything.
And esp not rendering sometimes is not related with offsets at all.

**Comment by mildnsj** [comment]
no, we won't be updating the offsets in the future, because it makes "opensource" pointless.
also, esp not rendering it has nothing to do with offsets.
